tournaments/index.js

```diff
@@ -19,7 +19,7 @@ function usersToNames(users) {
 function createTournamentGenerator(generator, args, output) {
 	var Generator = TournamentGenerators[toId(generator)];
 	if (!Generator) {
-		output.sendReply(generator + " is not a valid type.");
+		output.errorReply(generator + " is not a valid type.");
 		output.sendReply("Valid types: " + Object.keys(TournamentGenerators).join(", "));
```

Comment:
He said to make the above line an `errorReply` **as well**, which means this line should also be an `errorReply`.